frontend/src/semantic-ui/site/modules/tab.overrides

```diff
@@ -1,3 +1,8 @@
 /*******************************
         User Overrides
 *******************************/
+.ui.tab.active {
+    border: none;
+    border-radius: 0;
+    background-color: #F1F2F5;
+}
```

Comment:
Could you check if this will affect My Profile page?

Reply:
If it does, I will adjust it